Documentation.md

```diff
@@ -1582,7 +1582,53 @@ See below to discover used fields and see [reading section](#read) to discover s
 </pre>
 
 
-# GraphQL API
+# GraphQL API <span class="tag">alpha</span>
+
+## Getting started on GraphQL
+
+GraphQL is our new API All new resources will only be exposed in this new API.
+We are currently migrating old resources from Rest API to GraphQL.
+Please note that we are in alpha version, resources are pretty stable but
+authentication method will change in the next months.
+
+One of the power o f GraphQL API is to allow you retrieve many resources in one HTTP call.
```

Comment:
"o f" => of

Reply:
Instead of making one HTTP call by resource
C'est quoi qui te gêne dans la phrase ?

Reply:
ça me donnait une impression de lourdeur à la lecture c'est tout